### PR TITLE
Save initial options with preserveSelected

### PR DESCRIPTION
**Pull request by maxvyaznikov**
Without this, after selectbox loaded and user clicked on some of option, another options is removed.

### CODE REVIEW

File: src/js/classes/AjaxBootstrapSelectList.js [hunk 1]
```diff
@@ -45,6 +45,22 @@ var AjaxBootstrapSelectList = function (plugin) {
...
+        initial_options.push({
+            value: value,
+            text: $option.text(),
+            'class': $option.attr('class') || '',
+            data: $option.data() || {},
+            preserved: false,
+            selected: false
+        });
+    });
+    this.cacheSet(/*query=*/'', initial_options)
```
Comment (truckingsim) [inline src/js/classes/AjaxBootstrapSelectList.js]: Add a semi-colon at the end of this line.  JSHint fails without it.

**Comment by truckingsim** [comment]
@maxvyaznikov since verb is breaking and they haven't released an update, and travis is breaking due to this; I'm going to run the tests manually on my machine.  Once it passes everything and I confirm that it fixes the issues I'll probably merge it unless @markcarver has any issues with it.

**Comment by truckingsim** [comment]
@maxvyaznikov looks like the build is passing now (can do a full build not just testing).  Will test the fix later today and hopefully get this merged in soon.

**Comment by maxvyaznikov** [comment]
Yeah, thanks.

**Comment by truckingsim** [comment]
@maxvyaznikov out of curiousity do you have to trigger a change event on the select for it to notice the selected option?  I had to do that in my testing.  This definitely fixes the issue of it storing the selected option correctly though and thanks for that.

@markcarver merging this in because it definitely fixes half the problem with initial options not being selected properly.

**Comment by maxvyaznikov** [comment]
@truckingsim, in my case, options for select is loaded with page in html, and only after that JS load to start fun with options.

**Pull request merged**